### PR TITLE
Update to toggle style

### PR DESCRIPTION
**Pull request by aguscruiz**
Updated the toggle design to match Figma.

### CODE REVIEW

File: src/components/toggle/toggle.svelte [hunk 1]
```diff
@@ -241,6 +241,7 @@
       .thumb {
         --thumb-offset: var(--checked-thumb-offset);
         color: var(--checked-color);
+        --thumb-disabled-color: var(--leo-color-schemes-on-primary);
```
Comment (AlanBreck) [inline src/components/toggle/toggle.svelte:244]: This breaks the themeability of the toggle:
SCR-20241126-jnjx

Comment (aguscruiz) [reply src/components/toggle/toggle.svelte:244]: Ugh :(

Comment (aguscruiz) [reply src/components/toggle/toggle.svelte]: @fallaciousreasoning @AlanBreck  shouldn't `--leo-color-schemes-on-primary` change depending on the color that is chosen as primary? Or how should I do that? 


Btw I just realized this is also broken on the working version? If I customize the color there it doesn't show it on the thumb
https://nala.bravesoftware.com/?path=/docs/components-toggle--docs&globals=theme:dark

Comment (aguscruiz) [reply src/components/toggle/toggle.svelte]: Ok, I think I fixed it now.

I can change the color to whatever I want and it doesn't break on the default colors

image

image

**Review by AlanBreck — APPROVED** [review]

**Pull request merged**